protected/views/practice/index.php

```diff
@@ -22,12 +24,14 @@
           </h2>
         </div>
         <div class="large-4 column">
-          <?php $form = $this->beginWidget('CActiveForm', array(
-              'id' => 'practice-search-form',
-          )); ?>
-          <?php echo CHtml::textField('search-term', @$_POST['search-term'],
-              array('placeholder' => 'Enter search query...')); ?>
-          <?php $this->endWidget(); ?>
+            <?php $form = $this->beginWidget('CActiveForm', array(
+                'id' => 'practice-search-form',
+                'method' => 'get',
```

Comment:
Does lowercase actually work for HTTP verbs?

Reply:
Lowercase attributes seem to be the norm in HTML: https://www.w3schools.com/html/html_forms.asp